src/app/containers/StoryPromo/index.jsx

```diff
@@ -28,8 +28,9 @@ const StoryPromoImage = ({ imageValues, lazyLoad }) => {
   const ratio = (height / width) * 100;
   const originCode = getOriginCode(path);
   const locator = getLocator(path);
-  const srcset = createSrcset(originCode, locator, width);
-
+  const imageResolutions = [70, 95, 144, 183, 240, 320, 480, 624];
+  const srcset = createSrcset(originCode, locator, width, imageResolutions);
+  const sizes = topStory ? '(min-width: 600px) 33vw, 100vw' : '33vw';
```

Comment:
The images are roughly 33vw untill `1008px` not 600px. After 1008 theyre around 237px wide, not 100vw. Can we set this to be a fixed value?
A 30,000px wide screen still has 237px wide images after 1008, its not connected to vw

Reply:
@dr3  `600px` is the breakpoint at which the first story transitions to a top-story. I'll add a media query for screen sizes larger than`1008px`.